.github/workflows/build.yml

```diff
@@ -25,7 +25,7 @@ env:
 jobs:
   # This workflow contains a single job called "Xaml-Style-Check"
   Xaml-Style-Check:
-    runs-on: windows-latest
+    runs-on: windows-latest-large
```

Comment:
This shouldn't need the large runner, it's just running a cli tool only. We really don't want to use the large runner if we can help it, it has a limit to how many concurrent things can be run on it (and it costs more).